packages/pds-ember/tests/dummy/app/components/docs/button/button.hbs

```diff
@@ -1,6 +1,7 @@
 <section>
   <h2>&lt;Pds::Button&gt;</h2>
-  <Pds::ButtonSet>
+
+  <div style="display: flex; gap: 16px;">
```

Comment:
I tried doing this as a "replacement" for the old ButtonSet component so that the story examples using it would still display ok but obviously it's a no-no (linter doesn't like it). What should I do instead?

Reply:
Would be ok to just delete the div entirely and not worry about the Button examples not having proper spacing? Or else perhaps leave the old ButtonSet component there but just remove its story and add a note in the files where it's used that it shouldn't be used? (I worry that it would still get used by accident in the 2nd case.)

Or any other ideas on how to handle this?

Reply:
`<Pds::Button>`s in structure are inline which allows us to drop the replacement `<div>`s you've set entirely, indeed. I would favour this option, keeps the code clean and has a minimal visual impact (spacing).